operations/node_management/Rebuild_NCNs/Power_Cycle_and_Rebuild_Nodes.md

```diff
@@ -161,50 +166,50 @@ Run these commands on the rebuilt node.
 
         ```bash
         ncn# dig +short ${NODE}.can
-        10.103.8.11
+        10.101.5.141
         ```
 
 2. Confirm the output from the dig command matches the interface.
 
    * If the IP addresses match, proceed to the next step. If they do not match, continue with the following sub-steps.
 
         ```bash
-        ip addr show vlan007
+        ip addr show bond0.can0
         ```
 
         Example Output:
 
         ```screen
-        15: vlan007@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc noqueue state UP group default qlen 1000
-            link/ether b8:59:9f:2b:2f:9e brd ff:ff:ff:ff:ff:ff
-            inet 10.103.8.11/24 brd 10.103.8.255 scope global vlan007
-               valid_lft forever preferred_lft forever
-            inet6 fe80::ba59:9fff:fe2b:2f9e/64 scope link
-               valid_lft forever preferred_lft forever
+        8: bond0.can0@bond0: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9000 qdisc noqueue state UP group default qlen 1000
+           link/ether a4:bf:01:65:6a:aa brd ff:ff:ff:ff:ff:ff
+           inet 10.101.5.141/26 brd 10.101.5.191 scope global bond0.can0
+              valid_lft forever preferred_lft forever
+           inet6 fe80::a6bf:1ff:fe65:6aaa/64 scope link
+              valid_lft forever preferred_lft forever
         ```
 
-        1. Change the IP address for `vlan007` if necessary.
+        1. Change the IP address for `bond0.can0` if necessary.
 
             ```bash
-            vim /etc/sysconfig/network/ifcfg-vlan007
+            vim /etc/sysconfig/network/ifcfg-bond0.can0
             ```
 
-            Set the `IPADDR` line to the correct IP address with a `/24` mask. For example, if the correct IP address is `10.103.8.11`, the line should be:
+            Set the `IPADDR` line to the correct IP address with a `/26` mask. For example, if the correct IP address is `10.101.5.141`, the line should be:
```

Comment:
If this is for 1.2 then vlan007 should be cmn instead of can.

The netmask is going to depend on what subnets were provided when the system was installed or upgraded.   (I.e. the inputs to csi or the SLS update tool)

Reply:
For most of our internal systems, this is a /25 but customers could give us /26, /24.

Reply:
thank you - fixed

Reply:
Also, would it make sense to add something like this to the doc to help determine what the cmn netmask was set to during install or upgrade?
```
# kubectl get cm metallb -n metallb-system -o json | jq -r '.data.config' | yq r -P - | grep -B 1 "name: customer-management$"
      - 10.101.7.64/26
    name: customer-management
```

Reply:
Or there is an SLS query we could have them run.  I'll try to dig that up.

Reply:
```
cray sls networks list --format json | jq -r '.[] | select(.Name == "CMN") | .ExtraProperties.CIDR'
```

Reply:
fixed - thank you